src/components/SubscriptionListItem.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+	<ListItem :title="podcastData?.title ?? sub.url"
```

Comment:
On my setup, I cannot compile your code due to this line:
```
Errors compiling template:

  invalid expression: Unexpected token '.' in

    podcastData?.title ?? sub.url

  Raw expression: :title="podcastData?.title ?? sub.url"
```

But if I change it to `:title="getTitle(sub)"` and create a new function `getTitle(sub) { return this.podcastData?.title ?? sub.url }` it works. (doesn't work with `:title="this.podcastData?.title ?? sub.url"` either)

Do you have any idea what the reason behind this error might be? Looks like a strange Vue bug...

Reply:
Hmm very strange. I did update some packages, could you try run `npm install` again to see if that resolves it?

Reply:
Thanks for testing it out though!

I've changed to use methods everywhere instead. Could probably be solved but I don't want to jump into that rabbit hole right now.